### PR TITLE
Spiders buff and small change to critters

### PR DESCRIPTION
**Pull request by Jackrost**
This PR direct to change spiders and make them more big threat:
*) damage and HP were buffed. They now deal little poison damage.
*) No slipping!
*) can now pull stuff (items and crited people)
*) spiders can spawn webs that slowdown enemies and preys.

Another change: 
*) spiders now have separate event. You can tweak their count spawn.
*) critters spawns now don't depend on chaos level (No more 20-30 mice spawn)

This PR conflict with another PR #13292

**Media**
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame

:cl: Jack Rost
- tweak:  spiders have more damage and hp now, no slipping and can pull stuff around
- tweak: critters spawns now don't depend on chaos level
- add: spider web. Spiders now can spawn web that slowdown anyone expect spiders

### CODE REVIEW

File: Resources/Locale/en-US/actions/actions/spider.ftl [hunk 1]
```diff
@@ -0,0 +1,2 @@
+spider-web-action-name = Spider Web
+spider-web-action-description = Spawns web that slow your prey down.
```
Comment (AJCM-git) [inline Resources/Locale/en-US/actions/actions/spider.ftl:2]: Spawns **a** web that **slows***

**Review by AJCM-git — CHANGES_REQUESTED** [review]

File: Content.Server/Spider/SpiderSystem.cs [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            var coords = transform.Coordinates;
+            if (!grid.GetTileRef(coords).Tile.IsEmpty)
+            {
+                var ents = grid.GetLocal(coords);
+                bool net = false;
+                if (ents.All(x => !IsTileBlocked(x,ref net)))
+                {
+                    notBlocked = true;
+                    if (!net)
+                        EntityManager.SpawnEntity("SpiderWeb", transform.Coordinates);
```
[the remark this answers is not in the document]
Comment (Jackrost) [reply Content.Server/Spider/SpiderSystem.cs:72]: > use proxy method Spawn() and dont use prototypes ids like this, either move it to the component or use a tag. 

Do you mean create var in component as Datafield? What a type of this var need to be - string?

File: Content.Client/Spider/SpiderVisualsSystem.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+using Content.Shared.Spider;
+using Robust.Client.GameObjects;
+
+namespace Content.Client.Spider
+{
+
+    public sealed class SpiderVisualsSystem : VisualizerSystem<SpiderWebVisualsComponent>
```
[the remark this answers is not in the document]
Comment (Jackrost) [reply Content.Client/Spider/SpiderVisualsSystem.cs:7]: How can I do that if sprite name is generating? Generic visualizers expect me to put true/false as data as shown on exapmle
image

File: Content.Server/Spider/SpiderSystem.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (AJCM-git) [reply Content.Server/Spider/SpiderSystem.cs:72]: Yeah, a data field like
```
        [DataField("prototype", customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
        public string Prototype = "proto";
```

**Pull request closed without merging**